runtime/doc/eval.txt

```diff
@@ -2295,13 +2295,15 @@ v:version	Vim version number: major version times 100 plus minor
 			:if has("nvim-0.2.1")
 <
 
-				*v:vim_did_enter* *vim_did_enter-variable*
-v:vim_did_enter	0 during startup, 1 just before |VimEnter|.
+						*v:virtnum* *virtnum-variable*
+v:virtnum	Virtual line number for the 'statuscolumn' expressions.
+		A negative number indicates the number of virtual lines above
+		the start of the buffer line |v:lnum|, zero for the first text
+		row of v:lnum, and postive for the wrapped part of a line.
```

Comment:
Is this positive number always 1?

Reply:
Currently yeah, don't really think it's worth re-evaluating statuscolumn for each line wrap.